src/components/searchDisplay.tsx

```diff
@@ -0,0 +1,84 @@
+
+import { AccountDisplay } from '@src/components/accountDisplay';
+import { SearchResultsItem } from '@src/components/searchResultsItem';
+import { Account, AccountFields} from '@src/generated/sobs';
+import { Button, Form, Input } from 'antd';
+import * as React from 'react';
+import { hot } from 'react-hot-loader';
+import { generateSelect } from 'ts-force';
+const FormItem = Form.Item;
+
+interface SearchDisplayProps {
+    setAccount(account): void;
+}
+
+interface SearchDisplayStates {
+    searchValue: string;
+    searchResults: any;
+}
+
+export class SearchDisplay extends React.Component<SearchDisplayProps, SearchDisplayStates> {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            searchValue: '',
+            searchResults: null,
+        };
+    }
+
+    public updateSearchValue = (evt) => {
+        this.setState({
+          searchValue: evt.target.value,
+        });
+      }
+
+    public runSearch = () => {
+        Account
+        .retrieve(`SELECT ${generateSelect(Object.values(Account.FIELDS))} 
+        FROM Account WHERE Name LIKE '%${this.state.searchValue}%'`)
+        .then((accs) => this.setState({searchResults: accs}));
+    }
+
+    public render() {
+        return (
+            <div>
+                <Form layout='inline' style={{clear: 'both'}}>
+                    <FormItem>
+                        <Input 
+                            id='search-box'
+                            value={this.state.searchValue}
+                            onChange={evt => this.updateSearchValue(evt)}
```

Comment:
this is redundant.   Because you aren't changing the parameter in anyway, just pass in the function (instead of calling `updateSearchValue` in a inline function).

Reply:
this is the equivent of 
```
function(evt){
    this.updateSearchValue(evt);
}
```